app/src/main/java/com/example/nvdovin/weatherapp/model/CurrentForecast.java

```diff
@@ -2,7 +2,7 @@
 
 import com.example.nvdovin.weatherapp.database.model.WeatherData;
 
-public class ForecastNow {
+public class CurrentForecast {
```

Comment:
This class contains Forecast infor for a city, a name like : CityForcecast seems to be better, what do you think about?